environment.yml

```diff
@@ -1,4 +1,5 @@
-
+# For creating conda environments.
+# Please keep in sync with .github/workflows/build.yml .
```

Comment:
moved to the project root because it was out of sight, and wanted something visible telling us to keep these two files in sync